### PR TITLE
Improved map data collection

### PR DESCRIPTION
**Pull request by roll**
- fix https://github.com/okfn/opendataday/issues/233

### CODE REVIEW

**Comment by roll** [comment]
@jobarratt 
@amercader 
Please take a look

I've fixed the script and switched to using the first sheet from the Google Doc. I think it makes sense os the editor just needs to move an active sheet to be the first sheet in the doc.

File: assets/js/map.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-var url = "https://spreadsheets.google.com/feeds/list/1cV43fuzwy2q2ZKDWrHVS6XR4O8B01eLevh4PD6nCENE/4/public/full?alt=json";
+var url = "https://spreadsheets.google.com/feeds/list/1p8BdQAIGYpgsu-oCHxrCIwvKOa9y3lnV-BYSTbCa-zI/1/public/full?alt=json";
+// TODO: recover when ODD2020 has been moved to the first position
+// var url = "https://spreadsheets.google.com/feeds/list/1cV43fuzwy2q2ZKDWrHVS6XR4O8B01eLevh4PD6nCENE/1/public/full?alt=json";
```
Comment (roll) [inline assets/js/map.js]: Has to be updated before deploying

**Comment by roll** [comment]
It's updated.

@StephenAbbott 
Hi, after merging you need to move ODD2020 to the first place among the doc's sheets

**Comment by StephenAbbott** [comment]
Thanks @amercader and @roll! I'm just updating the site to link to the live 2020 registration form now and will also move the ODD2020 to the first place on the doc's sheets

**Pull request merged**